bin/set-sqfs-links.sh

```diff
@@ -56,6 +56,14 @@ ceph_initrd="$(find ${WEB_ROOT}/ephemeral/data/ceph -name "*initrd*" -printf '%T
 ceph_kernel="$(find ${WEB_ROOT}/ephemeral/data/ceph -name "*.kernel" -printf '%T@ %p\n' | sort -n | tail -1 |  cut -f2- -d" ")"
 ceph_squashfs="$(find ${WEB_ROOT}/ephemeral/data/ceph -name "*.squashfs" -printf '%T@ %p\n' | sort -n | tail -1 |  cut -f2- -d" ")"
 
+test -z $k8s_initrd && echo "ERROR: k8s initrd not found in ${WEB_ROOT}/ephemeral/data/k8s" && exit 1
```

Comment:
These `echo`s should redirect to stderr, since they're errors.

Reply:
Fair. Fixing.